airflow-core/src/airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -33,6 +33,7 @@ import { useLocalStorage } from "usehooks-ts";
 
 import type { DagRunState, DAGWithLatestDagRunsResponse } from "openapi/requests/types.gen";
 import DeleteDagButton from "src/components/DagActions/DeleteDagButton";
+import { FavoriteDagButton } from "src/components/DagActions/FavoriteDagButton";
```

Comment:
We also probably need a new 'filter' in case the user only want to see, list and filter his favorite dags. (A button to toggle/untoggle filter on `is_favorite True/False`)